Outlines/ethersjs.md

```diff
@@ -0,0 +1,15 @@
+# EthersJs
+
+- What is EthersJs?
```

Comment:
@jacobcreech As part of this part let's make sure we open with the problem that Etherjs is solving, or if not the problem, the gap it fills, or the alternative it provides.